src/dialects/mssql/index.js

```diff
@@ -37,6 +37,7 @@ export class MssqlDialect extends AbstractDialect {
       type: true,
       using: false,
       where: true,
+      include: true,
```

Comment:
You should add the include option in the abstract index as well;
https://github.com/sequelize/sequelize/blob/fe3f470194e41677c25504910cb5458a70382529/src/dialects/abstract/index.ts#L82-L92
and 
https://github.com/sequelize/sequelize/blob/fe3f470194e41677c25504910cb5458a70382529/src/dialects/abstract/index.ts#L195-L205

Reply:
Added now